src/internal/github/git.go

```diff
@@ -8,41 +8,55 @@ import (
 	"strings"
 )
 
-func GetTags(repositoryUrl string) ([]string, error) {
-	log.Printf("Getting tags for repository %s", repositoryUrl)
-
-	var buf bytes.Buffer
-	var bufErr bytes.Buffer
-	cmd := exec.Command("git", "ls-remote", "--tags", "--refs", repositoryUrl)
-	cmd.Stdout = &buf
-	cmd.Stderr = &bufErr
-
-	if err := cmd.Run(); err != nil {
-		log.Printf("Could not get tags for repository %s: %s", repositoryUrl, bufErr.String())
-		return nil, err
-	}
+func parseTagsFromStdout(lines []string) ([]string, error) {
+	exec.LookPath("")
+	tags := make([]string, 0, len(lines))
 
-	tags := make([]string, 0)
-	for _, line := range strings.Split(buf.String(), "\n") {
+	for _, line := range lines {
 		if !strings.Contains(line, "refs/tags/") {
 			continue
 		}
 
 		fields := strings.Fields(line)
 		if len(fields) != 2 {
-			return nil, fmt.Errorf("could not parse tags for %s: tags are in wrong format", repositoryUrl)
+			return nil, fmt.Errorf("Invalid format for tag '%s', expected two fields", line)
 		}
 
 		ref := fields[1]
 		if !strings.HasPrefix(ref, "refs/tags/") {
-			return nil, fmt.Errorf("could not parse tags for %s: tags are in wrong format", repositoryUrl)
+			return nil, fmt.Errorf("Invalid format for tag '%s', expected 'refs/tags/' prefix", line)
 		}
 
 		tag := strings.TrimPrefix(ref, "refs/tags/")
+		if tag == "" {
+			return nil, fmt.Errorf("Invalid format for tag '%s', no version provided", line)
+		}
+
 		tags = append(tags, tag)
 	}
 
-	log.Printf("Found %d tags for repository %s", len(tags), repositoryUrl)
+	return tags, nil
+}
 
+func GetTags(repositoryUrl string) ([]string, error) {
```

Comment:
Preferably I'd like GoDoc comments for atleast each exported function from a package. Even if it seems self explanatory, every little helps new developers approaching this.